src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1614,7 +1614,7 @@ private function addSchedulerSection(ArrayNodeDefinition $rootNode, callable $en
             ->children()
                 ->arrayNode('scheduler')
                     ->info('Scheduler configuration')
-                    ->{$enableIfStandalone('symfony/scheduler', SchedulerTransportFactory::class)}()
+                    ->{$enableIfStandalone('symfony/scheduler', Schedule::class)}()
```

Comment:
We need to use a class that has no dependencies on other components.
Here, if we require Scheduler without Messenger, you will get a PHP error.